Packs/CrowdStrikeFalconX/TestPlaybooks/CrowdStrikeFalconX_Test.yml

```diff
@@ -450,11 +450,6 @@ tasks:
             value:
               simple: csfalconx.resource.environment_description
             iscontext: true
-      - - operator: isNotEmpty
-          left:
-            value:
-              simple: csfalconx.resource.submit_url
-            iscontext: true
```

Comment:
I removed this since if the submitted resource was not a URL, then this will always return null, and I had to upload a resource that would have a threat score, and I couldn't find one that is of type URL (all the ones I would upload would give an empty threat score), I only found one of type file that would give me a threat score, therefore I deleted this check to not spend too much time on this.